Assets/MRTK/SDK/Experimental/StepSlider/StepSlider.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.UI;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.UI
+{
+    [AddComponentMenu("MRTK/SDK/Experimental/StepSlider")]
+    public class StepSlider : PinchSlider, IMixedRealityPointerHandler
```

Comment:
Please make sure to [add summary comments ](https://microsoft.github.io/MixedRealityToolkit-Unity/version/releases/2.3.0/Documentation/Contributing/CodingGuidelines.html#function--method-summary-headers) to all public classes, methods, and fields. 

From our guidelines: "All public classes, structs, enums, functions, properties, fields posted to the MRTK should be described as to its purpose and use...This ensures documentation is properly generated and disseminated for all all classes, methods, and properties."